polkadot/node/core/approval-voting/src/tests.rs

```diff
@@ -1428,68 +1383,6 @@ fn subsystem_rejects_approval_before_assignment() {
 	});
 }
 
-#[test]
-fn subsystem_rejects_assignment_in_future() {
```

Comment:
Checkes are now done in approval-distribution, so I moved the test there as well.